cmd/api-errors.go

```diff
@@ -888,6 +889,11 @@ var errorCodes = errorCodeMap{
 		Description:    "No matching replication rule found for this object prefix",
 		HTTPStatusCode: http.StatusBadRequest,
 	},
+	ErrReplicationDenyEditError: {
+		Code:           "XMinioReplicationDenyEdit",
+		Description:    "Replication config cannot be altered in cluster replication setup",
```

Comment:
Can we make this a bit clearer?

Maybe:
```suggestion
		Description:    "Cannot alter local replication config since this server is in a cluster replication setup",
```